critiquebrainz/frontend/external/bookbrainz_db/edition_group.py

```diff
@@ -0,0 +1,77 @@
+import uuid
+from brainzutils import cache
+from typing import List
+import sqlalchemy
+import critiquebrainz.frontend.external.bookbrainz_db as db 
+from critiquebrainz.frontend.external.bookbrainz_db import DEFAULT_CACHE_EXPIRATION
+from critiquebrainz.frontend.external.bookbrainz_db.identifiers import fetch_identifiers
+from critiquebrainz.frontend.external.bookbrainz_db.relationships import fetch_relationships
+
+
+def get_edition_group_by_bbid(bbid: uuid.UUID) -> dict:
```

Comment:
I see that you have a type hint here saying that this method takes a UUID object, but if I try it I get an error:
```py
In [9]: edition_group.get_edition_group_by_bbid(uuid.UUID("343ca0d1-cf7e-4a69-8701-97733d81c3b0"))
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
Input In [9], in <cell line: 1>()
----> 1 edition_group.get_edition_group_by_bbid(uuid.UUID("343ca0d1-cf7e-4a69-8701-97733d81c3b0"))

File /code/critiquebrainz/frontend/external/bookbrainz_db/edition_group.py:22, in get_edition_group_by_bbid(bbid)
     20 if not edition_group:
     21     return None
---> 22 return edition_group[bbid]

KeyError: UUID('343ca0d1-cf7e-4a69-8701-97733d81c3b0')
```

This is similar to the previous issue of upper-case letters in a UUID string.

In this case we should probably convert it to `str(bbid)` in the case that it's a UUID object.